bundles/AdminBundle/Controller/Admin/DataObject/DataObjectController.php

```diff
@@ -1289,7 +1289,10 @@ protected function updateIndexesOfObjectSiblings(DataObject\AbstractObject $upda
                         DataObject::OBJECT_TYPE_VARIANT,
                         DataObject::OBJECT_TYPE_FOLDER,
                     ]
-                ).'\') ORDER BY o_index LIMIT '. $updatedObject->getParent()->getChildAmount() .')
+                ).'\') ORDER BY o_index LIMIT '. $updatedObject->getParent()->getChildAmount([
+                            DataObject::OBJECT_TYPE_OBJECT,
+                            DataObject::OBJECT_TYPE_VARIANT
```

Comment:
```suggestion
                            DataObject::OBJECT_TYPE_VARIANT,
                            DataObject::OBJECT_TYPE_FOLDER
```

Reply:
I think we may need to add Folder in as well, as it was a default value already?
![image](https://user-images.githubusercontent.com/58705994/234109514-f701bb4f-d2ec-4ce9-8305-54065c29af8b.png)